ui/components/input/base/_index.scss

```diff
@@ -290,12 +290,42 @@
   z-index: 1;
   white-space: nowrap;
   position: relative;
-  top: -2px;
-  margin-left: 5px;
+  top: -0.05rem;
+  margin-#{$left}: 5px;
 }
 
 .slds-control-action__container svg {
   cursor: pointer;
 }
 
 /**********************************/
+
+
+ul.slds-dropdown-list {
+    position: absolute;
+    border: 2px solid #EEEEEE;
```

Comment:
I'm guessing this is the new stuff you added - can you please fix the colors in this block of code too?